pressbooks/pressbooks
# chore: update heroicons to include new ones

This PR adds new heroicons to the heroicons font and update the css mapping file.

AdjustmentsHorizontal.svg
Calendar.svg
ChevronDown.svg
ChevronUp.svg
ChevronUpDown.svg
Clock.svg
EllipsisHorizontal.svg
EllipsisVertical.svg
Funnel.svg


### How to test

1.  Pull this branch and double check all icons are ok on the PB sidebar menu interface
2. Try to use any of the new icons on any dashboard for instance
